docs/content/developer/getting-started/simple-token-transfer.mdx

```diff
@@ -0,0 +1,450 @@
+│  │ Version: 1                                                                                                                   │
+│  │ Digest: AqFmepZJmqEMi2qPYscZf1X4vZtXs3oUgSg2vn2X7CDz                                                                         │
+│  │ Modules: my_token                                                                                                            │
+│  └──                                                                                                                            │
+╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭───────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ Balance Changes                                                                                   │
+├───────────────────────────────────────────────────────────────────────────────────────────────────┤
+│  ┌──                                                                                              │
+│  │ Owner: Account Address ( 0xedeca9d6897cf86917e45f5c5cd596da48a5c4e3e1d9425aee42bed37c9ee44d )  │
+│  │ CoinType: 0x2::iota::IOTA                                                                      │
+│  │ Amount: -14444400                                                                              │
+│  └──                                                                                              │
+╰───────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+</details>
+
+## Simple Token Transfer UI
+
+Let’s now set up the frontend application using the [IOTA dApp kit](../../ts-sdk/dapp-kit/index.mdx). This kit provides a collection of tools and components that simplify integrating IOTA-based smart contracts into your web applications. By leveraging the dApp kit, you can efficiently build user interfaces that enable users to interact with the blockchain, such as sending transactions, querying contracts, and displaying data.
+
+The dApp kit is designed to streamline the connection between your frontend and the IOTA blockchain, handling important tasks like wallet integration, transaction signing, and communication with your deployed contract. If you’re new to the dApp kit, you can consult its documentation for comprehensive instructions on how to set it up and use its features to create a smooth frontend experience.
+
+In this section, we’ll walk you through the process of configuring the frontend and linking it to the smart contract we created earlier. By the end, you’ll have a fully functional dApp that enables users to interact with your IOTA-based token transfer contract directly from the frontend.
+
+### Setup 
+
+Run the following the command to setup the initial IOTA dApp.
+
+```bash
+pnpm create @iota/create-dapp
+```
+
+This will setup an initial IOTA dApp including all the necessary library.
```

Comment:
````suggestion
This will set up an initial IOTA dApp, including all the necessary dependencies. However, keep in mind that you will still need to install the dependencies by running the following command:

```npm2yarn
pnpm i
```
````

Since it's a getting-started tutorial, we need to provide all the instructions, as simple/obvious as they may sound.